### ISSUE TITLE
[FEATURE] - Visual distinction between labelled & unlabelled bookmarks

### ISSUE THREAD

**Issue by mlntr** [alefragnani/vscode-bookmarks#614]
There should be an easy distinction between unlabelled bookmarks and labelled bookmarks

For example, by default, labelled bookmarks can be light blue, while unlabelled ones remain the current blue:

- Here are two simple bookmarks (first one not labelled second one is labelled)
![image](https://github.com/alefragnani/vscode-bookmarks/assets/17552475/b0fffa5b-9eab-4bf1-984f-81354cfc1ee7)

- With this small change, users can easily tell which is (not) labelled
![image](https://github.com/alefragnani/vscode-bookmarks/assets/17552475/666b586f-331a-4f10-997f-e025b2cbf0e3)

**Comment by alefragnani** [comment]
Hi @mlntr ,

I wouldn't use color/transparency to differentiate between regular and labeled bookmarks, as the user can customize the color of bookmarks. Instead, I think a good alternative would be using different icons.

Labeled bookmarks shows a 🏷️  in the Command Palette and adds a ✏️ in the Side Bar. That's something that I would like to _merge_ and use the same identity on both locations. And based on your request, the same icon would be used in the gutter as well. I would vote for the 🏷️  icon.

What do you think?

**Comment by phazei** [comment]
I agree a color difference wouldn't be great. I'd suggest a similar icon that is the same bookmark shape, just with the pencil inside it, or just a letter "a" on top of it.
